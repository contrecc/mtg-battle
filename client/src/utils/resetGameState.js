export default function resetGameState(state, props) {
  return {
    bothStrong: false,
    bothWeak: false,
    winner: false,
    winningCard: '',
    roundCompleted: false,
    pickedCard: ''
  };
}
