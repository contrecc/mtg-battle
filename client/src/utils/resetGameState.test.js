import resetGameState from './resetGameState';

describe('resetGameState', () => {
  it('should return the updated state', () => {
    expect(resetGameState({}, {})).toMatchSnapshot();
  });
});
