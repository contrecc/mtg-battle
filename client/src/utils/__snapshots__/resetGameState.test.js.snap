// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resetGameState should return the updated state 1`] = `
Object {
  "bothStrong": false,
  "bothWeak": false,
  "roundCompleted": false,
  "winner": false,
  "winningCard": "",
}
`;
